### ISSUE TITLE
Incorrect Text for Item Rewards upon RoE Completion 

### ISSUE THREAD

**Issue by Era-Lusiphur** [topaz-next/topaz#2425]
**_I have:_**

- [x] searched existing issues (https://github.com/topaz-next/topaz/issues) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

Current client on current release. Complete any Records of Eminence objective with an item reward and it gives text stating you receive an amount of gil that's actually the itemid.

This example is for a Copper Voucher, itemid 8711. Confirmed this behavior with completing 2 other RoEs.
![Lusiphur_2021 01 12_165227](https://user-images.githubusercontent.com/61239975/104380604-41a73880-54f9-11eb-95b6-37d64960f2dd.png)

**Comment by wrenffxi** [comment]
TextIDs shifted in client update today.  Your client version is probably out of sync with server.
Current client version is 30201229_1 on release branch.

**Comment by Era-Lusiphur** [comment]
[**@wrenffxi**](https://github.com/wrenffxi) commented on [Jan 12, 2021, 5:14 PM EST](https://github.com/topaz-next/topaz/issues/2425#issuecomment-759070094 "2021-01-12T22:14:55Z - Replied by Github Reply Comments"):
> TextIDs shifted in client update today. Your client version is probably out of sync with server.  
> Current client version is 30201229\_1 on release branch.

I did a File Check for FFXI prior to logging in. I checked with @kaincenteno  and he said he has Canaria set to only allow current or greater versions to connect. He did server update around Noon Eastern today. Bootloader didn't report a mismatch on login.

Edit: Somehow I am on 30201204_0! Interesting. Will close this.
